hmac_authentication/tests/test_hmac_authentication.py

```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from hmac_authentication import exceptions
 from hmac_authentication import hmacauth
+from hmac_authentication import exceptions
+
+import hashlib
 import io
 import unittest
 
+
 HmacAuth = hmacauth.HmacAuth
+AuthenticationResult = hmacauth.AuthenticationResult
+AuthenticationResultCodes = hmacauth.AuthenticationResultCodes
```

Comment:
Minor, but you can delete a few lines if you change the import above to `from hmac_authentication.hmacauth import HmacAuth, AuthenticationResult, AuthenticationResultCodes`.